services/QuillLMS/client/app/assets/styles/evidence-tool.scss

```diff
@@ -348,6 +348,20 @@
   }
 }
 
+.tutorial-video-container {
+  height: 750px;
+  background-color: #F8F9F9;
+  .video-section video {
+    padding-top: 50px;
```

Comment:
I think for the most part, we try to do spacing in multiples of 4 (so I'd use 48px here instead of 50).